polaris-patterns/src/components/ResourceDetailsLayout/ResourceDetailsLayout.tsx

```diff
@@ -0,0 +1,37 @@
+import type {PropsWithChildren} from 'react';
+import React from 'react';
+import {VerticalStack, HorizontalGrid, AlphaCard} from '@shopify/polaris';
+
+export type ResourceDetailsLayoutProps = PropsWithChildren;
+
+export const ResourceDetailsLayout: React.FunctionComponent<PropsWithChildren> & {
+  PrimarySection: typeof PrimarySection;
+  SecondarySection: typeof SecondarySection;
+  Card: typeof Card;
+} = function ResourceDetailsLayout({children}: PropsWithChildren) {
```

Comment:
I left `Page` outside this component so builders will still have to compose `Page` themselves. This can enable them to add things like Banners between the Page header and layout. Alternatively, we could make this more strict and have the builder pass in `Page` like a slot